CodeStandards.md

```diff
@@ -0,0 +1,139 @@
+# ServUO Coding Standards
+
+## General Conventions
+
+### Use tabs for indents. 

```

Comment:
At the moment current `.editorconfig` states spaces are the current standard. I'm not sure whether the tab vs spaces discussion has ever happened in the past, so maybe project maintainers can provide some insight here.

In any case I would try to keep coding standards compliant with editor settings.